pose-detection/src/blazepose_tfjs/blazepose_test.ts

```diff
@@ -103,49 +84,112 @@ describeWithFlags('BlazePose', ALL_ENVS, () => {
   });
 });
 
+async function expectModel(
+    image: HTMLImageElement, segmentationImage: HTMLImageElement,
+    modelType: poseDetection.BlazePoseModelType) {
+  const startTensors = tf.memory().numTensors;
+
+  // Note: this makes a network request for model assets.
+  const detector = await poseDetection.createDetector(
+      poseDetection.SupportedModels.BlazePose, {
+        runtime: 'tfjs',
+        modelType,
+        enableSmoothing: true,
+        enableSegmentation: true,
+        smoothSegmentation: false
+      });
+
+  const beforeTensors = tf.memory().numTensors;
+
+  const result = await detector.estimatePoses(
+      image,
+      {runtime: 'tfjs', maxPoses: 1, flipHorizontal: false} as
+          poseDetection.BlazePoseTfjsEstimationConfig);
+  const xy = result[0].keypoints.map((keypoint) => [keypoint.x, keypoint.y]);
+  const xyz = result[0].keypoints3D.map(
+      (keypoint) => [keypoint.x, keypoint.y, keypoint.z]);
+
+  const segmentation = result[0].segmentation;
+  const maskValuesToLabel =
+      Array.from(Array(256).keys(), (v, _) => segmentation.maskValueToLabel(v));
+  const mask = segmentation.mask;
+  const actualBooleanMask = imageToBooleanMask(
```

Comment:
In demos we use soft segmentation mask. Why binary mask in test?

Reply:
It's rounded in the mediapipe test https://github.com/google/mediapipe/blob/master/mediapipe/python/solutions/pose_test.py#L171